packages/template-retail-react-app/app/commerce-api/auth.js

```diff
@@ -393,95 +437,27 @@ class Auth {
 
         const {id_token, enc_user_id, customer_id} = response
         let customer = {
-            authType: 'guest',
+            authType: Auth.USER_TYPE.GUEST,
             customerId: customer_id
         }
         // Determining if registered customer or guest
         if (id_token.length > 0 && enc_user_id.length > 0) {
-            customer.authType = 'registered'
+            customer.authType = Auth.USER_TYPE.REGISTERED
         }
         return customer
     }
-    /**
-     * Stores the given auth token.
-     * @private
-     * @param {string} token - A JWT auth token.
-     */
-    _saveAccessToken(token) {
-        this._authToken = token
-        if (this._onClient) {
-            this._storage.set(tokenStorageKey, token)
-        }
-    }
-
-    /**
-     * Stores the given usid token.
-     * @private
-     * @param {string} usid - Unique shopper Id.
-     */
-    _saveUsid(usid) {
-        this._usid = usid
-        if (this._onClient) {
-            this._storage.set(usidStorageKey, usid)
-        }
-    }
-
-    /**
-     * Stores the given enc_user_id token. enc = encoded
-     * @private
-     * @param {string} encUserId - Logged in Shopper reference for Einstein API.
-     */
-    _saveEncUserId(encUserId) {
-        this._encUserId = encUserId
-        if (this._onClient) {
-            this._storage.set(encUserIdStorageKey, encUserId)
-        }
-    }
-
-    /**
-     * Stores the given oid token.
-     * @private
-     * @param {string} oid - Unique organization Id.
-     */
-    _saveOid(oid) {
-        this._oid = oid
-        if (this._onClient) {
-            this._storage.set(oidStorageKey, oid)
-        }
-    }
 
     /**
      * Removes the stored auth token.
      * @private
      */
     _clearAuth() {
-        this._customerId = undefined
-        this._authToken = undefined
-        this._refreshToken = undefined
-        this._usid = undefined
-        this._encUserId = undefined
-        if (this._onClient) {
-            this._storage.remove(tokenStorageKey)
-            this._storage.remove(refreshTokenStorageKey)
-            this._storage.remove(refreshTokenGuestStorageKey)
-            this._storage.remove(usidStorageKey)
-            this._storage.remove(encUserIdStorageKey)
-            this._storage.remove(dwSessionIdKey)
-        }
-    }
-
-    /**
-     * Stores the given refresh token.
-     * @private
-     * @param {string} refreshToken - A JWT refresh token.
-     */
-    _saveRefreshToken(refreshToken, type) {
-        this._refreshToken = refreshToken
-        const storeageKey =
-            type === 'registered' ? refreshTokenStorageKey : refreshTokenGuestStorageKey
-        if (this._onClient) {
-            this._storage.set(storeageKey, refreshToken, {expires: REFRESH_TOKEN_COOKIE_AGE})
-        }
+        this._storage.delete(tokenStorageKey)
```

Comment:
In #505, I saw a TODO to restore _customerId. Shouldn't we clear _customerId from memory when clearing auth?

Reply:
I did a global search for `_customerId`, it turns out it's dead code, it's not being used at all.

Reply:
Ahhh. In that case, I'lll probably reintroduce it in the PR I'm working on but that's not a blocker for this.